app/uk/gov/hmrc/cbcr/connectors/DESConnector.scala

```diff
@@ -95,7 +94,7 @@ trait DESConnector extends RawResponseReads with HttpErrorFunctions {
     implicit val writes = SubscriptionRequest.subscriptionWriter
     Logger.info(s"Create Request sent to DES")
     http.POST[SubscriptionRequest, HttpResponse](s"$serviceUrl/$cbcSubscribeURI", sub).recover {
-      case e: HttpException => HttpResponse(e.responseCode, responseString = Some(e.message))
+      case e: HttpException => HttpResponse.apply(status = e.responseCode, body = e.message)
```

Comment:
Is there a reason you've made apply explicit?

Reply:
Because of deprecation warnings and I was pointed to use the apply method which has a few versions for different casses.